src/core/crypto.rs

```diff
@@ -1,47 +1,10 @@
-use num_bigint::{BigInt, Sign};
```

Comment:
it appears that we're no longer using the `num-bigint` dependency anywhere, so I think it can be removed from `Cargo.toml` (h/t `cargo udeps`)